espresso/environment/2_espresso_liveness_test.go

```diff
@@ -139,9 +139,10 @@ func TestE2eDevNetWithEspressoEspressoDegradedLiveness(t *testing.T) {
 // degraded state.
 //
 // The criteria for this test is as follows:
-//	Requirement: Liveness:
-//    The rollup should continue to run, [to] post Espresso confirmations
-//    within 10 seconds of each rollup block produced by the sequencer.
+//
+//		Requirement: Liveness:
+//	   The rollup should continue to run, [to] post Espresso confirmations
+//	   within 10 seconds of each rollup block produced by the sequencer.
```

Comment:
Shall we say _11 seconds_ instead?

Reply:
Addressed in 531670b35ef4b2c48a5199616d6d59b10dcdbfc4. With some other fixes.